verticapy/connection/write.py

```diff
@@ -67,6 +90,34 @@ def delete_connection(name: str) -> bool:
     bool
         True if the connection was deleted,
         False otherwise.
+
+    Example
+    -------
+    .. ipython:: python
+        :suppress:
+
+        from verticapy.connection import new_connection
+
+        new_connection({"host": "10.20.110.10",
+                        "port": "5433",
+                        "database": "vertica_eon",
+                        "password": "vertica",
+                        "user": "dbadmin"},
+                        name = "My_New_Vertica_Connection")
+
+    Display the available connections, delete the "My_New_Vertica_Connection" 
+    connection, and confirm that it no longer appears in the available
+    conenctions:
+
+    .. ipython:: python
+
+        from verticapy.connection import available_connections, delete_connection
+
+        available_connections()
```

Comment:
should you print results before and after?

Reply:
They do print in the rendered output